DECLARING_DEPENDENCIES.md

```diff
@@ -69,19 +69,18 @@ select compatible versions.
 
 Conflicts can occur when multiple artifacts from a single library are
 part of a dependency tree, and Maven resolves different artifacts from
-that library to versions from different releases. Using a BOM fixes
-this problem because a BOM dictates consistent versions for all
-artifacts from a library.
+that library to versions from different releases. The libraries-bom
+dictates consistent versions for all GCP orbit artifacts.
 
-You can use a BOM like this—this example is for `google-cloud-bom`:
+You use the BOM like this:
```

Comment:
The purpose of this section in the overall doc is to show how BOMs are imported in general. It is a section lower down that makes recommendations on which BOMs to use.

Reply:
I found myself coming here to find info about how to add the BOM to other projects, and not finding what I needed. It's a simpler story and usually best to say just use the libraries-bom.